optimade/server/models/structures.py

```diff
@@ -1,14 +1,521 @@
 from optimade.server.models.entries import EntryResourceAttributes, EntryResource
+from optimade.server.models.util import conlist
+from pydantic import Schema, BaseModel
+from typing import List, Optional
+
+
+class Specie(BaseModel):
```

Comment:
I had to check this myself but I believe  [the singular is species](https://grammarist.com/usage/species/)
```suggestion
class Species(BaseModel):
```

Reply:
It is, but I think one of pymatgen or ASE use specie/species for clarity but honestly I don't mind either way.